pandas/core/ops.py

```diff
@@ -788,22 +845,17 @@ def add_methods(cls, new_methods):
 
 # ----------------------------------------------------------------------
 # Arithmetic
-def add_special_arithmetic_methods(cls, arith_method=None,
-                                   comp_method=None, bool_method=None):
+def add_special_arithmetic_methods(cls):
     """
     Adds the full suite of special arithmetic methods (``__add__``,
     ``__sub__``, etc.) to the class.
 
     Parameters
     ----------
-    arith_method : function (optional)
-        factory for special arithmetic methods:
-        f(cls, op, special)
-    comp_method : function (optional)
-        factory for rich comparison - signature: f(cls, op, special)
-    bool_method : function (optional)
-        factory for boolean methods - signature: f(cls, op, special)
+    cls : class
+        special methods will be defined and pinned to this class
     """
+    arith_method, comp_method, bool_method = _get_method_wrappers(cls)[2:]
```

Comment:
i'd rather you now do this an instead use ``_, _, ...`` for the return values its more obvious